rero_ils/modules/acq_order_lines/jsonschemas/acq_order_lines/acq_order_line-v0.0.1.json

```diff
@@ -19,8 +19,8 @@
     "pid",
     "acq_account",
     "acq_order",
-    "amount",
-    "description"
+    "document",
```

Comment:
How the document is required? Is that means that I have to catalog the document before buying it?

Reply:
Yes, we need to catalog the document before order it.